doc/nrf/protocols/dect/index.rst

```diff
@@ -0,0 +1,144 @@
+
+Security
+   Possibility of hiding the network using AES-128 encryption and integrity protection.
+   DECT NR+ communication can be encrypted and the network itself hidden for enhanced security.
+
+High-speed
+   DECT NR+ reaches data rates up to 3.4 Mbps, depending on modulation.
+   DECT NR+ allows selecting the modulation scheme for high-speed communication suitable for various applications.
+
+Comparison of DECT NR+ and LTE
+******************************
+
+While both DECT NR+ and LTE address different needs, DECT NR+ offers distinct advantages in specific areas, as summarized by the following table:
+
+.. table:: Feature comparison of DECT NR+ and LTE
+
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Feature                               | DECT NR+                          | LTE                        |
+   +=======================================+===================================+============================+
+   | License                               | License-exempt 1.9 GHz            | Requires licenses (varies) |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | IP based                              | No                                | Yes                        |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Coverage                              | Shorter, up to 3 km               | Wider, greater than 10 km  |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Performance in congested environments | Better                            | Poor                       |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Network flexibility                   | Star or mesh networks             | Primarily carrier-provided |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Power Consumption                     | Lower                             | Higher                     |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Data Rate (max)                       | 3.4 Mbps                          | Varies (typically higher)  |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Latency                               | Lower                             | Higher                     |
+   +---------------------------------------+-----------------------------------+----------------------------+
+   | Security                              | Strong encryption (AES-128)       | Encryption available       |
```

Comment:
AES is also used in LTE

Reply:
If LTE and DECT provide the same amount of security, then there's no point in making this comparison.
Do you think we should remove this row?

Reply:
I'd suggest to  remove this comparison